### PR TITLE
Specify max dependence versions

### LINKED ISSUE #1020: Old Kiwix desktop versions in ppa repository, blocker for inclusion in Labdoo / Ubuntu (report from Mastodon user)
Report from a Mastodon user:

> Hi, Labdoo / Ubuntu was updated from kiwix.2.1 to kiwix.2.2. Kiwix.2.2 requires libkiwix11 as a dependency, but during the update the version libkiwix10 13.00-jammy was installed and the whole thing doesn't want to work together. Actually this should not happen because we install kiwix from a ppa. But you are not in it. Do you have the bug on your screen? Is a patch planned? Thank you.
>
> PS Labdoo uses 22.04 LTS. Kiwix.2.2 needs libkiwix11, but 22.04 LTS latest patch still comes with libkiwix10 13.00-jammy.

This user doesn't have a GitHub account, so couldn't make a report themselves. Is this fixable?

### PR DESCRIPTION
**Pull request by kelson42**
Fixes #1020

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -6,8 +6,8 @@ Build-Depends: debhelper-compat (= 13),
  pkg-config,
  qtbase5-dev,
  qtwebengine5-dev,
- libkiwix-dev (>= 11.0.0~)
-Standards-Version: 4.5.0
```
[the remark this answers is not in the document]
Comment (kelson42) [reply debian/control:10]: yes, will reintroduce

**Review by veloman-yunkan — APPROVED** [review]

File: debian/control [hunk 2]
```diff
@@ -6,7 +6,8 @@ Build-Depends: debhelper-compat (= 13),
  pkg-config,
  qtbase5-dev,
  qtwebengine5-dev,
- libkiwix-dev (>= 11.0.0~)
+ libkiwix-dev (>= 13.0.0), libkiwix-dev (<< 14.0.0)
+ libzim-dev (>= 9.0.0), libzim-dev (<< 10.0.0)
```
Comment (mgautierfr) [inline debian/control:10]: Debian packaging seems to not understand `libkiwix-dev (<< 14.0.0)`. (See https://github.com/kiwix/kiwix-desktop/actions/runs/8053475026/job/21995971665)

Maybe this should be a simple `<` ?

**Review by mgautierfr — CHANGES_REQUESTED** [review]

File: debian/control (hunk 2 above)
Comment (veloman-yunkan) [reply debian/control:10]: I checked the [documentation](https://www.debian.org/doc/debian-policy/ch-relationships.html#syntax-of-relationship-fields) during my review. `<<` seems to be the only valid syntax:

> The relations allowed are `<<`, `<=`, `=`, `>=` and `>>` for strictly earlier, earlier or equal, exactly equal, later or equal and strictly later, respectively.

Comment (veloman-yunkan) [reply debian/control:10]: I think the problem is with the missing comma after `libkiwix-dev (<< 14.0.0)`

Comment (kelson42) [reply debian/control]: Fixed!

**Pull request merged**